core/client-internal/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -69,6 +70,9 @@
   /** Constant error message for block ID not cached. */
   private static final String BLOCK_ID_NOT_CACHED =
       "The block with ID {} could not be cached into Alluxio storage.";
+  /** Error message for cache collision. */
+  private static final String BLOCK_ID_EXISTS_SO_NOT_CACHED =
```

Comment:
I will refactor the two error messages in this class to `ErrorMessage` in a separate PR to master. (Ticket: https://alluxio.atlassian.net/browse/ALLUXIO-1826)